src/api/src/logger/index.ts

```diff
@@ -1,5 +1,6 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
+import { QueryOptions } from 'winston';
```

Comment:
Generic interface should not expose concrete implementation query options type.

In this case it would be the best to create an interface which would implement the required subset of `winston` `QueryOptions` interface to work.

Reply:
Changes pushed.